### PR TITLE
Add example with multiple observables

### PR DESCRIPTION
**Pull request by FreyMo**

### CODE REVIEW

File: multiple-observables/StatusObserver.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+            // DistinctUntilChanged filters out observed values that are equal to their predecessors.
+
+            Observable.CombineLatest(statusObservables,
+                (lastStates) => lastStates.Any(status => status == Status.Error))
+                .DistinctUntilChanged()
+                .Subscribe(hasState => StatusError(hasState));
+                
+            Observable.CombineLatest(statusObservables,
+                (lastStates) => lastStates.All(status => status == Status.Ready))
```
Comment (TomaT3) [inline multiple-observables/StatusObserver.cs]: I recognized that it wouldn't work like it is now when you want to switch on/blink/off lights.

**Review by TomaT3 — CHANGES_REQUESTED** [review]

File: multiple-observables/StatusObserver.cs (hunk 1 above)
Comment (FreyMo) [reply multiple-observables/StatusObserver.cs]: Interesting. Functionally I did not make any changes. How should we proceed with this issue?

Comment (TomaT3) [reply multiple-observables/StatusObserver.cs]: I will show you how I would do it when I'm back home. After four more beers yesterday, I realized that I shouldn't turn off the light in two different methods which will be called in unknown order. So maybe the light will go off, but should stay on.

Comment (FreyMo) [reply multiple-observables/StatusObserver.cs]: Classic. Let's shift the discussion to monday. We should talk in person.

Comment (TomaT3) [reply multiple-observables/StatusObserver.cs]: With the new approach, that will work (I'm quite sure) we will get the cool switch expression back in code 😁

Comment (FreyMo) [reply multiple-observables/StatusObserver.cs]: Wow that's impressive! You should consider working as a professional software developer.

**Pull request merged**